test/integration/createTippy.test.js

```diff
@@ -179,8 +179,6 @@ describe('instance.destroy()', () => {
     jest.advanceTimersByTime(500);
 
     expect(spy).not.toHaveBeenCalled();
-
-    spy.mockRestore();
```

Comment:
Why was this removed?

Reply:
This spy is not used anymore during this test, so it's redundant.

Reply:
What if I add another spy later on in the file?

Reply:
I meant this particular test case. Other `it`s will work fine.

Cleaning up is usually good even if not needed, but I haven't seen manual restoring in other test cases 🙄

Reply:
tbh I don't understand properly, but I'll trust you're right..

> I haven't seen manual restoring in other test cases

Should that be another PR?

Reply:
Well, `jest` uses `spy.mockRestore()` in [their example](https://jestjs.io/docs/en/jest-object#jestspyonobject-methodname), so I guess we can leave it there.